data/json/monsters/fish.json

```diff
@@ -1162,31 +1162,62 @@
   {
     "id": "mon_mutant_salmon",
     "type": "MONSTER",
-    "name": { "str": "giant salmon" },
-    "description": "A mutated salmon, the size of a large dog and quite dangerous to the inexperienced angler.",
+    "name": { "str_sp": "giant salmon" },
+    "description": "A mutated salmon roughly the size of a person.  Quite dangerous to the inexperienced angler.",
     "default_faction": "mutant",
     "bodytype": "fish",
     "species": [ "FISH" ],
     "volume": "62500 ml",
     "weight": "81500 g",
-    "hp": 180,
-    "speed": 150,
+    "hp": 110,
+    "armor": { "bash": 4, "cut": 3, "bullet": 2 },
+    "speed": 175,
     "material": [ "flesh" ],
     "symbol": "S",
     "color": "white_magenta",
-    "aggression": 100,
-    "morale": 105,
+    "aggression": -20,
+    "morale": 80,
     "melee_skill": 4,
     "melee_dice": 2,
     "melee_dice_sides": 6,
-    "melee_damage": [ { "damage_type": "cut", "amount": 2 } ],
+    "melee_damage": [ { "damage_type": "cut", "amount": 4 } ],
     "dodge": 5,
-    "luminance": 0,
     "harvest": "mutant_fish",
     "dissect": "dissect_fish_sample_small",
     "reproduction": { "baby_egg": "egg_fish", "baby_count": 2, "baby_timer": 150 },
     "baby_flags": [ "AUTUMN" ],
-    "flags": [ "SEES", "HEARS", "SMELLS", "SWIMS", "AQUATIC" ]
+    "anger_triggers": [ "HURT" ],
+    "flags": [ "SEES", "HEARS", "SWIMS", "AQUATIC" ]
+  },
+  {
+    "id": "mon_mutant_salmon_huge",
+    "type": "MONSTER",
+    "name": { "str_sp": "huge salmon" },
+    "description": "A huge mutant salmon grown almost to the size of an alligator.",
+    "default_faction": "mutant",
+    "bodytype": "fish",
+    "species": [ "FISH" ],
+    "volume": "312 L",
+    "weight": "265 kg",
+    "hp": 225,
+    "armor": { "bash": 5, "cut": 5, "bullet": 3 },
+    "speed": 175,
+    "material": [ "flesh" ],
+    "symbol": "S",
+    "color": "white_magenta",
+    "aggression": 0,
```

Comment:
```suggestion
    "color": "white_magenta",
```

Reply:
Yeah I know I've seen the error. I'm ignoring it for now until I figure out something for the next mutant size, then I'll handle it. Thanks for catching it, though!